modules/cnf-fast-event-notifications-api-refererence.adoc

```diff
@@ -289,3 +292,58 @@ $ oc logs -f linuxptp-daemon-cvgr6 -n openshift-ptp -c cloud-event-proxy
    }
 }
 ----
+
+== /api/cloudnotifications/v1/<resource_address>/CurrentState
+
+[discrete]
+=== HTTP method
+
+`GET api/cloudNotifications/v1/<resource_address>/CurrentState`
+
+[discrete]
+==== Description
+
+Returns the current state of `os-clock-sync-state`, `ptp-clock-class-change`, and `lock-state` for the cluster node. 
+
+* `os-clock-sync-state` notifications describe the host operating system clock synchronization state. Can be in `LOCKED` or `FREERUN` state.
+* `ptp-clock-class-change` notifications describe the current state of the PTP clock class.
+* `lock-state` notifications describe the current status of the PTP equipment lock state. Can be in `LOCKED`, `HOLDOVER` or `FREERUN` state.
+
+.Query parameters
+|===
+| Parameter | Type
+
+| `<resource_address>`
+| string
+|===
+
+.Example os-clock-sync-state current state API response
+[source,json]
+----
+{
+  "specversion": "0.3",
+  "id": "4f51fe99-feaa-4e66-9112-66c5c9b9afcb",
```

Comment:
Here is the example for lock-state
 {"id": "c1ac3aa5-1195-4786-84f8-da0ea4462921",
   "type": "event.sync.ptp-status.ptp-state-change",
   "source": "/cluster/node/cnfde7.ptp.lab.eng.bos.redhat.com/sync/ptp-status/lock-state",
   "dataContentType": "application/json",  
   "time": "2023-01-10T02:41:57.094981478Z",
   "data": {
              "version": "v1",
              "values": [
                  { "resource": "/cluster/node/cnfde7.ptp.lab.eng.bos.redhat.com/ens5fx/master",
                    "dataType": "notification",
                    "valueType": "enumeration",
                    "value": "LOCKED"
                   },
                   {"resource": "/cluster/node/cnfde7.ptp.lab.eng.bos.redhat.com/ens5fx/master\",
                    "dataType": "metric",
                    "valueType": "decimal64.3",
                    "value": "29"
                    }
                    ]
            }
}

Reply:
and for clock class 
{"id": "064c9e67-5ad4-4afb-98ff-189c6aa9c205",
  "type": "event.sync.ptp-status.ptp-clock-class-change",
  "source": "/cluster/node/compute-1.example.com/sync/ptp-status/ptp-clock-class-change",
  "dataContentType": "application/json",
  "time": "2023-01-10T02:41:56.785673989Z",
  "data": {"version": "v1",
      "values": [{"resource": "/cluster/node/compute-1.example.com/ens5fx/master",
      "dataType": "metric",
      "valueType": "decimal64.3",
      "value": "165"
      }]
      }
 }